tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers.ts

```diff
@@ -46,12 +46,15 @@ const initialState: DebuggerState = {
       lastLoadedTimeInMs: null,
     },
     scrollBeginIndex: 0,
+    focusIndex: null,
```

Comment:
FYI, `executions` is getting quite fat. It might benefit from creating a sub-reducer.